packages/@azure/servicebus/data-plane/test/renewLock.spec.ts

```diff
@@ -363,7 +363,9 @@ async function testBatchReceiverManualLockRenewalHappyCase(
   );
 
   await delay(5000);
-  await receiver.renewLock(msgs[0]);
+  if (msgs[0].lockToken) {
```

Comment:
what exactly are we expecting/testing here?

Reply:
`renewLock(lockTokenOrMessage, options)`method takes either lockToken or message as a parameter. All the tests were using only message as a parameter so I updated some of the tests to use the lock token just to ensure that we have tests covering this feature.